### PR TITLE
Added Personal Page

### PR DESCRIPTION
**Pull request by yashgo0018**
## Description

Screenshot from 2024-04-09 15-16-11

Added Personal Page

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

#8

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address: buidlguidl.yashgoyal.eth

### CODE REVIEW

**Comment by technophile-04** [comment]
Thanks @yashgo0018 ! Added a small comment, also could you please fix lint warning because of which the GH-actions are failing. 

We could ignore the lint warning by `{/* eslint-disable-next-line @next/next/no-img-element */}` and use img tag instead of Next Image component just to avoid updating `next.config` and adding remote URL pattern their

File: packages/nextjs/app/builders/0x396E5C4EC6965b46Ff95d71f0cBF219d19DD5071/page.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"use client";
```
[the remark this answers is not in the document]
Comment (yashgo0018) [reply packages/nextjs/app/builders/0x396E5C4EC6965b46Ff95d71f0cBF219d19DD5071/page.tsx]: Thanks for this, I have updated the code now.

**Comment by yashgo0018** [comment]
I have fixed these issues in the new commit

File: packages/nextjs/public/social-symbols/discord.svg
Comment (technophile-04) [inline packages/nextjs/public/social-symbols/discord.svg:1]: We are not using this social icons right ? Let's remove them ?

Comment (yashgo0018) [reply packages/nextjs/public/social-symbols/discord.svg:1]: Didn't mean to push them, pushed them by mistake, removed them now.

**Review by technophile-04 — APPROVED** [review]
Awesome ! Thanks @yashgo0018 looking great !

**Pull request merged**